source/installguide/management-server/_pkg_repo.rst

```diff
@@ -99,19 +99,19 @@ repository to the file (replace "trusty" with "xenial" or "bionic" if it is the
 
 .. parsed-literal::
 
-   deb http://download.cloudstack.org/ubuntu focal |version|
+   deb https://download.cloudstack.org/ubuntu focal |version|
 
 We now have to add the public key to the trusted keys.
 
 .. parsed-literal::
 
-   wget -O - http://download.cloudstack.org/release.asc |sudo apt-key add -
+   wget -O - https://download.cloudstack.org/release.asc |sudo tee /etc/apt/trusted.gpg.d/cloudstack.asc
```

Comment:
Should this be, something like:
```
mkdir -p /etc/apt/keyrings
wget -O- https://download.cloudstack.org/release.asc | gpg --dearmor | sudo tee /etc/apt/keyrings/cloudstack.gpg > /dev/null
```
And for use for the repo, the usage is like or something similar:
```
echo deb [signed-by=/etc/apt/keyrings/cloudstack.gpg] https://download.cloudstack.org/<path to repo here> / > /etc/apt/sources.list.d/cloudstack.list
```

Reply:
Would that really be needed that way? I have tested the commands and worked on my environment.

Happy to change it though